tket/src/Circuit/basic_circ_manip.cpp

```diff
@@ -459,6 +466,18 @@ register_t Circuit::add_c_register(std::string reg_name, unsigned size) {
   return ids;
 }
 
+void Circuit::add_wasm_register() {
+  if (!wasm_added) {
+    Vertex in = add_vertex(OpType::WASMInput);
+    Vertex out = add_vertex(OpType::WASMOutput);
+    add_edge({in, 0}, {out, 0}, EdgeType::WASM);
+    WASMUID wuid = WASMUID();
+    wasmwire = wuid;
+    boundary.insert({wasmwire, in, out});
```

Comment:
This this the right way to add new elements to the boundary? Is there something I am missing?